docs/cmd/tkn_taskrun_describe.md

```diff
@@ -34,6 +34,8 @@ or
   -L, --last                          show description for last TaskRun
       --limit int                     lists number of TaskRuns when selecting a TaskRun to describe (default 5)
   -o, --output string                 Output format. One of: json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-file.
+      --status                        use status to get the status of a TaskRun
+      --steps                         show the status of steps
```

Comment:
can we please mention it needs to be specified with `--status`

Reply:
Sure.